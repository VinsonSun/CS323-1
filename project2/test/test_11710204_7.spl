struct Apple{
    int price;
};
int test_11710204_7()
{
  int a = 2;
  struct Apple apple;
  apple.price = apple + a;
  return a;
}

