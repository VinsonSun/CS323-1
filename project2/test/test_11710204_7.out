Error type 7 at Line 8: binary operation on non-number variables
